src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -384,15 +386,15 @@ public static DynamoModel Start(StartConfiguration configuration)
             return new DynamoModel(configuration);
         }
 
-        protected DynamoModel(StartConfiguration configuration)
+        protected DynamoModel(StartConfiguration config)
         {
             ClipBoard = new ObservableCollection<ModelBase>();
             MaxTesselationDivisions = 128;
```

Comment:
It shouldn't be a constant.  It controls at runtime how precise the tessellation is.  Spelling error, though.